packaging/nuget/NServiceBus.Ninject.nuspec

```diff
@@ -13,9 +13,9 @@
     <description>The Ninject Container for the nservicebus</description>
     <releaseNotes></releaseNotes>
     <copyright>$copyright$</copyright>
-    <tags>nservicebus servicebus msmq cqrs publish subscribe</tags>
+    <tags>$tags$</tags>
     <dependencies>
-      <dependency id="NServiceBus" version="[5.0.0, 6.0.0)" />
+      <dependency id="NServiceBus" version="6.0.0-beta0001" />
```

Comment:
Didn't know that, @andreasohlund said to fix it to beta version, I must have misunderstood it. Then I have to fix the host as well